packages/ui/src/theme/overwrites/components/textarea.ts

```diff
@@ -0,0 +1,37 @@
+export const Textarea = {
+  baseStyle: {
+    width: '12.5rem',
+    height: '5rem',
+  },
+  variants: {
+    outline: {
```

Comment:
I think we should specify the `gray.300` as border color because by default its value is `gray.200`